assets/keymaps/macos/atom.json

```diff
@@ -1,21 +1,32 @@
+// Default Keymap (Atom) for Zed on MacOS
 [
   {
     "bindings": {
+      "ctrl-alt-cmd-l": "workspace::Reload",
       "cmd-k cmd-p": "workspace::ActivatePreviousPane",
       "cmd-k cmd-n": "workspace::ActivateNextPane"
     }
   },
   {
     "context": "Editor",
     "bindings": {
+      "ctrl-shift-l": "language_selector::Toggle",
+      "cmd-|": "pane::RevealInProjectPanel",
       "cmd-b": "editor::GoToDefinition",
       "alt-cmd-b": "editor::GoToDefinitionSplit",
       "cmd-<": "editor::ScrollCursorCenter",
       "cmd-g": ["editor::SelectNext", { "replace_newest": true }],
-      "ctrl-cmd-g": ["editor::SelectPrevious", { "replace_newest": true }],
+      "cmd-shift-g": ["editor::SelectPrevious", { "replace_newest": true }],
       "ctrl-shift-down": "editor::AddSelectionBelow",
       "ctrl-shift-up": "editor::AddSelectionAbove",
       "cmd-shift-backspace": "editor::DeleteToBeginningOfLine",
+      "cmd-k cmd-u": "editor::ConvertToUpperCase",
+      "cmd-k cmd-l": "editor::ConvertToLowerCase",
+      "alt-enter": "editor::Newline",
+      "cmd-j": "editor::JoinLines",
```

Comment:
AFAIK this conflicts with the default keybinding for toggling the bottom drawer, perhaps this shortcut was intentionally omitted prior to this PR?

I've no problem manually defining another key combination but this might cause some confusion for newer users who opt to use the Atom keybindings :)

Reply:
Nice catch. You're probably right, I've reverted that change here: https://github.com/zed-industries/zed/pull/13988